config/sync/core.entity_form_display.node.moj_pdf_item.default.yml

```diff
@@ -47,24 +48,25 @@ third_party_settings:
     group_series:
       children:
         - field_moj_series
+        - field_not_in_series
         - group_season_and_episode_number
         - group_release_date
       parent_name: ''
-      weight: 8
+      weight: 7
       format_type: fieldset
       region: content
       format_settings:
-        description: 'Select which series this episode belongs to. Then, select either a season and episode number <strong>or</strong> a date.'
+        description: 'Select the series this content belongs to. Then enter either a session and episode number or the date of release.'
```

Comment:
Minor copy query, the _date of release_ or _release date_

Reply:
Date of release is what Jo requested it be changed to.

Have just spotted another error: Then enter either a **session** and episode number (should be **season**)